Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/observables.tsv

```diff
@@ -0,0 +1,16 @@
+observableId	observableName	observableFormula	observableTransformation	noiseFormula	noiseDistribution
+observable_x_0ac	y_{0ac}	x_0ac	log	sigma	normal
```

Comment:
noiseFormula should be `noiseParameter1_$observableId`

Reply:
In the main (original) implementation of this problem, this is the case [1, 2]. My understanding of the PEtab format is that this should be equivalent to removing the `noiseParameter` column (which has `sigma` everywhere) from the measurements table, and replacing the `noiseParameter1_$observableId` formula with simply `sigma` everywhere.

[1] https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/blob/975f1989cc82b5c7fe09bb84c9275ed286ebaf33/Benchmark-Models/Blasi_CellSystems2016/observables_Blasi_CellSystems2016.tsv
[2] https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/blob/975f1989cc82b5c7fe09bb84c9275ed286ebaf33/Benchmark-Models/Blasi_CellSystems2016/measurementData_Blasi_CellSystems2016.tsv

Reply:
Yes, that works but the PEtab format is not defined like that https://petab.readthedocs.io/en/latest/documentation_data_format.html#id2

Reply:
That's allowed in PEtab. No problem.